pulp_smash/tests/rpm/api_v2/test_crud.py

```diff
@@ -0,0 +1,42 @@
+# coding=utf-8
+"""Tests that CRUD RPM repositories.
+
+For information on repository CRUD operations, see `Creation, Deletion and
+Configuration
+<http://pulp.readthedocs.io/en/latest/dev-guide/integration/rest-api/repo/cud.html>`_.
+"""
+from __future__ import unicode_literals
+
+from pulp_smash import utils
+from pulp_smash.tests.rpm.api_v2.utils import gen_repo
+from pulp_smash.tests.rpm.utils import set_up_module as setUpModule  # noqa pylint:disable=unused-import
+
+
+class CrudTestCase(utils.BaseAPICrudTestCase):
+    """CRUD a minimal RPM repository."""
+
+    @staticmethod
+    def create_body():
+        """Return a dict for creating a repository."""
+        return gen_repo()
+
+    @staticmethod
+    def update_body():
+        """Return a dict for creating a repository."""
+        return {'delta': {'display_name': utils.uuid4()}}
+
+
+class CrudWithFeedTestCase(utils.BaseAPICrudTestCase):
```

Comment:
For the sake of keeping the inheritance tree short. But, now that you've pointed this out to me, it doesn't seem so sensible any more. I'll update this.